Examples/Algorithms/Geant4/src/Geant4ParticleStatus.cpp

```diff
@@ -0,0 +1,25 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2024 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#include "ActsExamples/Geant4/Geant4ParticleStatus.hpp"
+
+#include <ostream>
+
+namespace ActsExamples {
+
+std::ostream &operator<<(std::ostream &os,
+                         Geant4ParticleStatus particleStatus) {
+  switch (particleStatus) {
+    case Geant4ParticleStatus::Undefined:
+      return (os << "undefined");
+    default:
+      return (os << static_cast<uint32_t>(particleStatus));
```

Comment:
should we have strings for these cases as well?

Reply:
Renamed to `ParticleOutcome` and added string prints.